engine/src/main/scala/cromwell/engine/workflow/tokens/JobExecutionTokenDispenserActor.scala

```diff
@@ -68,7 +70,15 @@ class JobExecutionTokenDispenserActor(override val serviceRegistryActor: ActorRe
   private def distribute(n: Int) = if (tokenQueues.nonEmpty) {
```

Comment:
[ToL] Would it not be a clearer design (though not any more correct) to synchronize on `tokenQueues`?

Reply:
Hopefully the actor `receive` is synchronizing access to the token queues for us - this is in case that turns out to not be true